package.json

```diff
@@ -20,7 +20,7 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^6.2.x",
     "babel-preset-es2015": "^6.24.1",
-    "backbone": "~1.1.2",
+    "backbone": "~1.3.3",
```

Comment:
There seem to be quite a bit of changes between these versions. When looking at the changelog did you see anything that makes sense to file in separate issues?